storage/client/client.go

```diff
@@ -262,13 +263,18 @@ func (c *StorageClient) Transactions(ctx context.Context, p apiTypes.GetConsensu
 			&t.Fee,
 			&t.Method,
 			&t.Body,
-			&code,
+			&t.Code,
+			&module,
+			&message,
```

Comment:
```suggestion
			&t.Module,
			&t.Message,
```

(here and below) Let's just scan it directly into the struct? I believe pgx will correctly scan nullable values into a double pointer. 

Unless there's a case where the tx is successful and the module/message are non-null and we do not want to show the module /message...

Reply:
+1, though this will change a little if/when we store all of these error fields into an `error` substructure.

Reply:
> Unless there's a case where the tx is successful and the module/message are non-null and we do not want to show the module /message...

Yeah. I was trying to avoid the empty strings.

Lemme see where things are after this `error` substructure refactor.

Reply:
Seems like we can either choose:

A. scan into empty error object and nullify it in case `code == 0`

B. construct error if `code != 0`.

I went with B but down with A.